frontend/src/components/eMIB/EmibTabs.jsx

```diff
@@ -65,7 +66,7 @@ class EmibTabs extends Component {
         </Helmet>
         <TabNavigation
           tabSpecs={TABS}
-          currentTab={0}
+          currentTab={this.props.currentTab}
```

Comment:
The naming of `currentTab` is a bit confusing because it doesn't actually always represent the current tab (the actual current tab is managed in state not props). It represents the `initialTab` or `startingTab` - which might be better names.

Reply:
Fixed in commit #https://github.com/code-for-canada/project-thundercat/pull/255/commits/aaf61e9e0a0f762a7bfabc49590c4d0d150b6cfc